frontend/src/components/display/HoursTable.js

```diff
@@ -0,0 +1,34 @@
+import React, {Component} from 'react';
+import Table from 'react-bootstrap/Table';
```

Comment:
these two imports can be the same line
import {Table, WhateverElement} from 'library';